### ISSUE TITLE
Binding & Helper inconsistency?

### ISSUE THREAD

**Issue by daveeel** [ManuelDeLeon/viewmodel#160]
Hi @ManuelDeLeon , here's a case
```jade
        .modal-header
          h4.modal-title Edit Node {{selectedNode.title}} ({{selectedNode._id}})
        .modal-body
          input(type='text' $b="value: selectedNode().title")
```
```ls
    selected-node: ->
      Nodes.find-one @selected-node-id!
```
The blaze style {{}} is perfectly fine, but the binding part gives error:
```
TypeError: Cannot use 'in' operator to search for 'false' in undefined
    at getValue (viewmodel.coffee:405)
    at getValue (viewmodel.coffee:373)
    at Object.getVmValue (viewmodel.coffee:419)
    at addBinding.autorun (bindings.coffee:55)
    at fun (viewmodel.coffee:248)
    at view.js:199
    at Function.Template._withTemplateInstanceFunc (template.js:457)
    at view.js:197
    at Object.Blaze._withCurrentView (view.js:538)
    at viewAutorun (view.js:196)
```
Using `input(type='text' $b="value: selectedNode.title")` gives the same error. Is it a bug?

**Comment by daveeel** [comment]
Digging a bit more, that solves my issue:
```jade
        .modal-header
          h4.modal-title Edit Node {{selectedNode.title}} ({{selectedNode._id}})
        .modal-body
          with selectedNode
            input(type='text' $b="value: this.title")
```
Is this the right intended use?

**Comment by daveeel** [comment]
Digging a bit more, that solves my issue:
```jade
        .modal-header
          h4.modal-title Edit Node {{selectedNode.title}} ({{selectedNode._id}})
        .modal-body
          with selectedNode
            input(type='text' $b="value: this.title")
```
Is this the right intended use?

**Comment by daveeel** [comment]
mm... finally I checked out your phonebook sample. I shall follow the .load pattern. 

Anyways, it doesn't quite make sense to bind a form / field directly to a DB record ...

**Comment by ManuelDeLeon** [comment]
Let's leave it open. I want to change it so the load order is guaranteed:

signal
share
mixin
load
Rest of the view model properties

The logic here is that the things to add to the view model get higher priority the closer they are to the view model.

**Comment by fvpDev** [comment]
+ 1

**Comment by ManuelDeLeon** [comment]
Fixed on v3.0.0
